src/content/images/teal-balloon.svg

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can we run this through https://jakearchibald.github.io/svgomg/

Reply:
Updated the image.  Thank you.